docs/ussri-blog/citeas-ussri-blog.md

```diff
@@ -0,0 +1,48 @@
+# CiteAs.org: Discovering and Improving software requests for citation
+
+- James Howison, [University of Texas at Austin](http://james.howison.name)
+- Heather Piwowar and Jason Priem, [Impact Story](https://impactstory.org)
+
+[CiteAs.org](https://citeas.org) links between pieces of software and their requested citations. Go from the name of a piece of software, its webpage URL, or a DOI directly to the machine-readable metadata (e.g., BibTex, Zotero auto-import) for the citation the author wants you to use. CiteAs.org is funded by the Digital Science program at the Sloan Foundation (Grant Number 8028), and conceived and developed by Heather Piwowar and Jason Priem at [ImpactStory](https://impactstory.org), together with [James Howison](http://james.howison.name) from the Information School at the University of Texas at Austin.
+
+**Great software work ⟶ Clear requests for citation ⟶ More visibility in publications ⟶ More credit ⟶ Better Software ⟶ Better Research**
+
+Software very rarely contains citation information, and when it does it is certainly not as obvious as it is in a paper. With a paper, if you have the paper you have the metadata right there in front of you. [Howison and Bullard, 2013](http://doi.org/10.1002/asi.23538) found that less than half of the times in which publications actually mentioned software were formal, traceable, citations.
+
+![image: a citation.](an_article_citation.png)
+
+But if you have a piece of software, you very likely don't have the metadata needed to cite it. In part this is because with software there is no standard place to "write" the information, but it's also because authors sometimes want their users to cite something other than the piece of software directly. Examples include citing a paper that introduces the software (or demonstrated its potential), a published software manual or book, a "software paper" created specifically as a citation target, or a benchmarking paper.
+
+Great work is being done to guide best practices (including the [FORCE11 Working Group on Software Citation](https://www.force11.org/group/software-citation-working-group)) which recommends always including a direct citation to the software itself, including version numbers---something key for reproducibility---in addition to papers. We don't disagree, but we think it's important to let the authors decide how their contribution should be acknowledged and to link users with those requests.
```

Comment:
Should the `something` here be `sometimes`?

Reply:
no, something is right.  Sometimes would also be ok, but is weaker.